measurement/measurement.go

```diff
@@ -744,3 +743,206 @@ func calculateRhumbDistance(origin []float64, destination []float64, radius *flo
 
 	return dist
 }
+
+// PointOnFeature takes a Feature and returns a point guaranteed to be on the surface of the feature.
+func PointOnFeature(f feature.Feature, properties map[string]interface{}, id string) (*geometry.Point, error) {
+	fs := []feature.Feature{}
+	fs = append(fs, f)
+	fc, err := feature.NewFeatureCollection(fs)
+	if err != nil {
+		return nil, err
+	}
+	return PointOnFeatureCollection(*fc, properties, id)
+}
+
+// PointOnFeatureCollection gets a feature collection and returns a point guaranteed to be on the surface of the feature.
+func PointOnFeatureCollection(fc feature.Collection, properties map[string]interface{}, id string) (*geometry.Point, error) {
```

Comment:
This function is exportable, so I believe that it's a good idea to include this function in the README as pointOnFeatureCollection and mark them as implemented.